tests/TestDocs.hs

```diff
@@ -306,8 +302,8 @@ displayAssertionFailure = \case
     "in rendered code for " <> decl <> ", bad link location for " <> target <>
     ": expected " <> T.pack (show expected) <>
     " got " <> T.pack (show actual)
-  WrongOrder _ before after' ->
-    "expected to see " <> before <> " before " <> after'
+  WrongOrder _ before' after' ->
```

Comment:
This change addresses shadowed name warnings from the `before` exported by Test.Hspec